### DISCUSSION TITLE
Spring boot issue sir

### DISCUSSION THREAD

**Discussion by Pasupula-Bhargavi** [akash-coded/spring-framework#73] [Q&A]
![Screenshot (435)](https://user-images.githubusercontent.com/103362221/167644059-8f657a8e-6cb6-47a3-91dd-5ff4f9f085d6.png)
![Screenshot (436)](https://user-images.githubusercontent.com/103362221/167644123-41065660-6b50-4e4e-8397-b1640d0d916f.png)

**Comment by akash-coded** [comment]
@Pasupula-Bhargavi We can set up a screen share some time tomorrow and get this ironed out if it hasn't been resolved yet.
